src/App.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-extraneous-dependencies */
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Books from './pages/Books';
+import Categories from './pages/Categories';
+import NotFound from './pages/NotFound';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <div className="container_1">
+        <Routes>
+          <Route path="/" element={<Books />} />
```

Comment:
- [ ] According to the requirements, the root `/` route should show a `<Home />` component which in turn will have the `<Books>` component and the `<Navbar />` component